pages/content/amp-dev/documentation/guides-and-tutorials/contribute/integrate-your-tech.md

```diff
@@ -29,7 +29,7 @@ There are 90+ ad providers who have added support for advertising related featur
 
 If you provide call tracking measurement services, your use case may be supported with the new [`amp-call-tracking`]({{g.doc('/content/amp-dev/documentation/components/reference/amp-call-tracking.md', locale=doc.locale).url.path}}) extension. This extension dynamically replaces a phone number in a hyperlink to enable call tracking, by executing a CORS request to substitute the number.
 
-To learn more about how this extension might work for you, please see an [AMP By Example]({{g.doc('/content/amp-dev/documentation/examples/components/amp-call-tracking.html', locale=doc.locale).url.path}}) or read about the [documentation]({{g.doc('/content/amp-dev/documentation/components/reference/amp-call-tracking.md', locale=doc.locale).url.path}}).
+To learn more about how this extension might work for you, please see an [AMP By Example]({{g.doc('/content/amp-dev/documentation/examples/documentation/amp-call-tracking.html', locale=doc.locale).url.path}}) or read about the [documentation]({{g.doc('/content/amp-dev/documentation/components/reference/amp-call-tracking.md', locale=doc.locale).url.path}}).
```

Comment:
does this need to be `/content/amp-dev/documentation/examples/components/amp-call-tracking.html